content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -501,6 +501,36 @@ As this brings Bevy closer to full support of Android, there isn't a need anymor
 
 ![iOS emulator running Bevy](ios%20emulator.png)
 
+## Enable Parallel Pipelined Rendering
+
+<div class="release-feature-authors">authors: @hymm, @james7132</div>
+
+![Trace with Pipelined Rendering](pipelined-rendering-trace.png)
+
+On multithreaded platforms bevy will now run significantly faster by running simulation and
+rendering in parallel. The renderer was rearchiteched in [bevy 0.6](https://bevyengine.org/news/bevy-0-6/#pipelined-rendering-extract-prepare-queue-render)
+to enable this, but the final step of actually running them in parallel was not done until
+now. There was a bit of tricky work to figure out around enabling running specific rendering
+systems on the main app thread instead of the rendering thread and making sure !Send
+resources aren't accessed from a different thread than they are initialized on.
+
+![Histogram of Many Foxes Frame Time](pipelined-rendering-histogram.png)
+
+In testing different bevy examples, the gains were typically in the 10% to 30% range. As seen in the above histogram the many foxes stress test gains 1.8ms.
```

Comment:
Also - can you be more specific about "gains `1.8ms`"? Do you mean "the stress test speeds up by an average of 1.8ms per frame"? I couldn't really tell from the image